pkg/velero/server/config.go

```diff
@@ -23,19 +23,19 @@ type ServerConfig struct {
 	// defaultBackupLocation will be defined outside of server config in DataProtectionApplication
 	// defaultBackupLocation                        string
 
-	// How often to ensure all Velero backups in object storage exist as Backup API objects in the cluster. This is the default sync period if none is explicitly specified for a backup storage location.
+	// How often (in minutes) to ensure all Velero backups in object storage exist as Backup API objects in the cluster. This is the default sync period if none is explicitly specified for a backup storage location.
 	// +optional
-	BackupSyncPeriod *time.Duration `json:"backup-sync-period,omitempty"`
-	// How long pod volume file system backups/restores should be allowed to run before timing out. (default 4h0m0s)
+	BackupSyncPeriod *int64 `json:"backup-sync-period,omitempty"`
+	//  How long pod volume file system backups/restores should be allowed to run before timing out. (default 4h0m0s)
```

Comment:
I would remove this extra space added in the description strins right at the start, I believe this is causing the insert of single quotes in CRD descriptions. Was this a desirable change ?

Reply:
this is for me to remember, all those field have wrong description, and I need to fix them. BUT prior to that, I want us to accept a solution